### PR TITLE
splitting tests into batches on Travis/CI

### PR DESCRIPTION
**Pull request by stopfstedt**
refs #4046

### CODE REVIEW

**Comment by stopfstedt** [comment]
further splitting integration tests from unit tests may be necessary, the 'non-acceptance' group is still failing repeatedly.

**Comment by stopfstedt** [comment]
success for now! had to restart the non-acceptance group two times, tho.

File: package.json [hunk 1]
```diff
@@ -138,5 +138,8 @@
     "lint:js",
     "lint:style"
   ],
-  "private": true
+  "private": true,
+  "dependencies": {
+    "ember-exam": "^1.0.0"
```
Comment (jrjohnson) [inline package.json]: This should go into `devDependencies`.

Comment (stopfstedt) [reply package.json]: good catch, fixed.

**Comment by stopfstedt** [comment]
i'm giving up on this, for now.

**Pull request closed without merging**